scripts/run.sh

```diff
@@ -49,45 +49,48 @@ echo AVALANCHE_LOG_LEVEL: ${AVALANCHE_LOG_LEVEL}
 # https://github.com/ava-labs/avalanchego/releases
 GOARCH=$(go env GOARCH)
 GOOS=$(go env GOOS)
-DOWNLOAD_URL=https://github.com/ava-labs/avalanchego/releases/download/v${VERSION}/avalanchego-linux-${GOARCH}-v${VERSION}.tar.gz
-DOWNLOAD_PATH=/tmp/avalanchego.tar.gz
+BASEFILE=/tmp/subnet-evm-runner
```

Comment:
Let's discuss switching to a standard approach of how we install avalanchego across different repos. We have a similar script in a few places now and maintaining them is getting difficult and fractured.